sdcm/fill_db_data.py

```diff
@@ -3237,29 +3235,28 @@ def _run_db_queries(self, item, session):
         for i in range(len(item['queries'])):
             try:
                 if item['queries'][i].startswith("#SORTED"):
-                    res = session.execute(item['queries'][i].replace('#SORTED', ''))
+                    res = self.run_query_with_retry(session=session, query=item['queries'][i].replace('#SORTED', ''))
```

Comment:
I'm not sure we want retries for all of the `_run_db_queries`, we don't usually have issues with those, and if they won't work at first shot, we do want to know about them

Reply:
done

Reply:
@fruch its a good question if we want to know about them or not.
Originally we wanted and reported some of them, but looks like the approach is to ask first for retries.
We can ask in one of the existing issues or in a new issue or in Slack engineering what should be the default behavior.

Reply:
We properly want to fix the driver and have support of exponential backoffs, and set it per session 

https://github.com/scylladb/python-driver/issues/91